servers/display/native_menu.cpp

```diff
@@ -30,7 +30,7 @@
 
 #include "native_menu.h"
 
-#include "scene/resources/image_texture.h"
+#include "scene/resources/image_texture.h" // encapsulation pragma: ignore // FIXME: legacy include.
```

Comment:
Use of `ImageTexture` can be changed to `Image` (which is code), but will be compact breaking since it's public API.